cypress/e2e/CMS/$AppModule.feature

```diff
@@ -0,0 +1,80 @@
+Feature: As a cms user I want to be able to create Article content type to promote Article
```

Comment:
can you rename this As a cms user I want to be able to create App module content type to deploy custom apps

Reply:
fixed